### PR TITLE
Performance Measurement Additional Calculations

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: drivers/performance_measurement/app/models/performance_measurement/details.rb [hunk 1]
```diff
@@ -512,38 +549,38 @@ def detail_hash
             'days_in_th_bed_details_in_period',
           ],
         },
+        # FIXME
```
Comment (awisema) [inline drivers/performance_measurement/app/models/performance_measurement/details.rb]: What needs fixing?

File: drivers/performance_measurement/app/models/performance_measurement/result_calculation.rb [hunk 2]
```diff
@@ -498,6 +498,71 @@ def time_to_move_in_median(detail, project: nil)
       )
     end
 
+    # FIXME
```
Comment (awisema) [inline drivers/performance_measurement/app/models/performance_measurement/result_calculation.rb]: Add a one-liner on what is unaddressed here?

**Review by awisema — APPROVED** [review]

File: drivers/performance_measurement/app/models/performance_measurement/details.rb (hunk 1 above)
Comment (eanders) [reply drivers/performance_measurement/app/models/performance_measurement/details.rb]: nothing now!

File: drivers/performance_measurement/app/models/performance_measurement/result_calculation.rb (hunk 2 above)
Comment (eanders) [reply drivers/performance_measurement/app/models/performance_measurement/result_calculation.rb]: These were to remind me where I needed to update code, which I did, but they didn't.

**Pull request merged**